### ISSUE TITLE
[Accessibility] Screen reader doesn't announce chip removal when a chip is removed via keyboard

### ISSUE THREAD

**Issue by opozo** [angular/components#12122]
#### Bug, feature request, or proposal:

When a chip [1] is removed via keyboard, screen reader doesn't announce it
[1] https://material.angular.io/components/chips/overview

#### What is the expected behavior?

When a chip is removed via keyboard, screen reader should announce chip removal.

#### What is the current behavior?

Screen reader doesn't announce chip removal when a chip is removed via keyboard.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

@angular/material: 6.3.0

**Comment by zarend** [comment]
Action item here is to update examples to announce when chip is removed using `LiveAnnouncer`. Also to update a11y documentation.
